.github/auto_assign.yml

```diff
@@ -0,0 +1,15 @@
+addReviewers: true
+
+addAssignees: false
+
+reviewers: 
+  - praveenscience
+  - vermakhushboo
+  - PrathibhaS
+  - priyadharshinir1894
+  - AnjaliSharma1234
```

Comment:
I guess we might need to remove `AnjaliSharma1234`, we just checked and she doesn't have write access.

Reply:
Done with this change Sir